docs/Contributing.asciidoc

```diff
@@ -97,6 +97,15 @@ requests for consideration or create an issue with a code change proposal.
 *** Do *not* get too creative with computational default values, e.g. `sub foo ($first, $second = rand($first)) {`
 *** Do *not* combine sub attributes with signatures (requires Perl 5.28+), e.g. `sub foo :lvalue ($first) {`
 
+* General
+
+** Let http://perltidy.sourceforge.net/[Perltidy] tidy perl files
+automatically:
+
+    tools/tidyall --git # tidy changed files
+    tools/tidyall --all # tidy all files
+    make tidy # to tidy also JavaScript files in the openQA project
+
```

Comment:
This looks inconsistent and redundant. The first sentence says "tidy perl" but then "make tidy" mentions JavaScript files. Our Makefile is well structured. Please don't pick details out of it. Just calling `make tidy` is what we should do and that is what https://github.com/os-autoinst/openQA/pull/6590/files#diff-4d6d1050121df87969e32770abd50500a1fbda9f136db4fbb374fe05a236f2c2R59 already mentions. so why this point "General" at all?